modules/client/src/main/kotlin/jp/co/soramitsu/iroha2/client/Iroha2Client.kt

```diff
@@ -139,6 +139,25 @@ open class Iroha2Client(
         }
     }
 
+    private var lastRequestedPeerIdx: Int? = null
+
+    // Round-robin load balancing
+    protected fun getTelemetryUrl() = getUrls().second
+
+    // Round-robin load balancing
+    protected fun getPeerUrl() = getUrls().first
+
+    private fun getUrls() = when (lastRequestedPeerIdx) {
```

Comment:
could this be a part of an implementation of some 'BalancingStrategy' interface?